src/pages/_app.tsx

```diff
@@ -35,9 +36,11 @@ function MoyeoApp({
       <QueryClientProvider client={queryClient}>
         <Hydrate state={dehydratedState}>
           <RecoilRoot>
-            <Layout>
-              <Component {...rest} />
-            </Layout>
+            <GlobalModal>
+              <Layout>
+                <Component {...rest} />
+              </Layout>
+            </GlobalModal>
```

Comment:
```suggestion
              <Layout>
                <Component {...rest} />
              </Layout>
            <GlobalModal />
```
`<GlobalModal>` 안에 다른 페이지를 렌더링하면 모달이 리렌더링될 때마다 다른 페이지도 리렌더링이 될 거 같은데 
`<GlobalModal>`을 독립적으로 분리를 하는건 어떨까요?